epoch-verifier/src/verifications/utils.ts

```diff
@@ -2,7 +2,7 @@ import { VerificationResult } from "../runner";
 
 type Predicate<T> = (elem: T) => boolean | VerificationResult
 
-export const SCALE_FACTOR = 10n ** 18n;
+export const SCALE_FACTOR = 10n ** 36n;
```

Comment:
Why the change here?

Reply:
I was testing the computation accuracy for different factors and forgot to change it since there's no much difference between 18n and 36n